README.md

```diff
@@ -0,0 +1,701 @@
+│    │   │      EVENT      │   │    │
+│    │   │                 │   │    │
+│    │   └─────────────────┘   │    │
+│    │                         │    │
+│    │   owner=organiser───┐   │    │
+│    │   │                 │   │    │
+│    │   │      EVENT      │   │    │
+│    │   │                 │   │    │
+│    │   └─────────────────┘   │    │
+│    │                         │    │
+│    │   owner=organiser───┐   │    │
+│    │   │                 │   │    │
+│    │   │      EVENT      │   │    │
+│    │   │                 │   │    │
+│    │   └─────────────────┘   │    │
+│    │                         │    │
+│    │                         │    │
+│    └─────────────────────────┘    │
+│                                   │
+└───────────────────────────────────┘
+```
+
+In the following diagram we can see how non-hierarchical ownership can be modeled with a photo sharing app. The app displays to the user a collection of photos, this isn't an "owned" collection as no peer has overall authority over it, each user will have their own photo collection. All they need is `read` authority for any published photos.
+
+```
+owner=anna─────────┐   owner=billie───────┐
+me:read            │   me:read            │
+│                  │   │                  │
+│      PHOTO       │   │      PHOTO       │
+│                  │   │                  │
+│                  │   │                  │
+└──────────────────┘   └──────────────────┘
+
+owner=claire───────┐   owner=diana────────┐
+me:read            │   me:read            │
+│                  │   │                  │
+│      PHOTO       │   │      PHOTO       │
+│                  │   │                  │
+│                  │   │                  │
+└──────────────────┘   └──────────────────┘
+```
+
+# System context assumptions
+
+We assume a system where resources can be referenced by unique identifiers (for example, via content hashes) and messages between peers are signed using public key cryptography. Optionally, messages can form append-only logs (by including sequence number and backlinks) so as to provide verifiable causal ordering and stricter capability condition restrictions.
+
+# Definitions
+
+**peer:** a user possessing knowledge of the private key for an ed25519 signing keypair; publishes operations and syncs document state with other peers  
+**owner:** the peer who created a document  
+**issuer:** the peer issuing authority via a capability  
+**receiver:** the peer receiving authority via a capability  
+**authorizer:** the peer authorizing a request or operation against a capability  
+**invoker** the peer attempting to use the authority a capability delegates (by publishing operations or syncing documents)  
+**document:** a uniquely identifiable resource which capabilities are associated with  
+**operation:** messages peers publish to the network containing changes to documents, require that suitable authority is demonstrated before being accepted by another peer  
```

Comment:
can messages be accepted "written to disk cos valid sig chain", but not accepted cos "failed capability verification"?

Reply:
Nope, only operations with the required authority are accepted.